models/_sector/nft/trades/nft_ethereum_trades_beta.sql

```diff
@@ -0,0 +1,31 @@
+{{ config(
+    schema = 'nft_ethereum',
+    alias ='trades_beta',
+    partition_by = ['block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['project','project_version','tx_hash','sub_tx_trade_id']
+    )
+}}
+-- (project, project_version, model)
+{% set base_models = [
+     ('archipelago',    'v1',   ref('archipelago_ethereum_base_trades'))
+    ,('superrare',    'v1',   ref('superrare_ethereum_base_trades'))
+    ,('foundation',    'v1',   ref('foundation_ethereum_base_trades'))
+] %}
+
+
+-- macros/models/sector/nft
+{{
+    enrich_trades(
+        blockchain='ethereum',
+        models=base_models,
+        transactions_model=source('ethereum','transactions'),
+        tokens_nft_model=ref('tokens_ethereum_nft'),
```

Comment:
i'm good with this, but curious your thoughts on leveraging the chain-specific spell vs. the broader spell and adding blockchain to join condition?

Reply:
I think it allows for a bit more flexibility when using the macro (esp during the initial development now)